src/google/protobuf/io/tokenizer_unittest.cc

```diff
@@ -341,7 +341,7 @@ inline std::ostream& operator<<(std::ostream& out,
 MultiTokenCase kMultiTokenCases[] = {
   // Test empty input.
   { "", {
-    { Tokenizer::TYPE_END       , ""     , 0,  0 },
+    { Tokenizer::TYPE_END       , ""     , 0,  0,  0 },
```

Comment:
Otherwise it triggers -Wmissing-field-initializers because there's another another field in that struct. All of the other initializers immediately following this in the file also explictly initialize this last member (which is end_column).